stable/artifactory-ha/values.yaml

```diff
@@ -1007,8 +1007,12 @@ nginx:
     ssl_prefer_server_ciphers   on;
     ## server configuration
     server {
-      listen {{ .Values.nginx.internalPortHttps }} ssl;
-      listen {{ .Values.nginx.internalPortHttp }} ;
+      {{- if .Values.nginx.https.enabled }}
+      listen {{ .Values.nginx.https.internalPort }} ssl;
+      {{- end }}
+      {{- if .Values.nginx.http.enabled }}
+      listen {{ .Values.nginx.http.internalPort }} ;
+      {{- end }}
```

Comment:
Thanks for catching this.

If i'm not mistaken though, this isn't backwards compatible to the old 1.3.0 values?

Reply:
You are **right**! I will fix this too. Thx for noticing!